### DISCUSSION TITLE
XIAO CAN Bus Expansion Board | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#911] [Q&A]
# XIAO CAN Bus Expansion Board | Seeed Studio Wiki

Getting started with the XIAO CAN Bus Expansion Board

https://wiki.seeedstudio.com/xiao-can-bus-expansion/

**Comment by TPPS999** [comment]
For anyone looking for working pinout for xiao esp32 s3 board following works for me
#define SPI_CS_PIN    44
#define SPI_MOSI_PIN  9
#define SPI_MISO_PIN  8  
#define SPI_SCK_PIN   7
